requirements.txt

```diff
@@ -23,3 +23,4 @@ numba
 emoji
 grapheme
 psutil==5.7.0
+lolcat
```

Comment:
LOL

Reply:
haha I guess pyphoon needs this to get the phases of the moon.